deltametrics/mobility.py

```
@@ -46,6 +46,10 @@ def check_inputs(chmap, basevalues, time_window, landmap=None):
         chmap = chmap.mask
```

If masks are now 2-D arrays, then checking and allowing a `mask.BaseMask` to be used for mobility is not going to work anymore. I think either this `elif` could be removed, or it should throw an error.

---

Ah yeah, I see that now, I've changed the statement in here so that any subclass of `BaseMask` as input will raise an error:
```
TypeError: Cannot input a Mask directly to mobility metrics. Use a list-of-masks instead.
```

---

also added a test to check this.